modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+When updating a cluster, it's essential to understand the available options and their implications. For detailed information on cluster updates and available options, you can refer to the command-line documentation using the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --help
+----
+
+When running this command, the following information is included:
+
+[source,terminal]
+----
+`Cluster-side guards include checks for release verification and upgradeable conditions. Again, it is usually best to give these conditions time to resolve, or to actively work to resolve them.  But if you decide to trigger the update regardless of these concerns, use --force, which is passed through to ClusterVersion's spec.desiredUpdate.force.`
+----
+It's important to understand the consequences of performing a forced update to determine if this decision outweighs the potential risks.
+If there is no other way to update a cluster, you can _force_ an update by using the `--force` option in the CLI, which allows you to forcefully update the cluster even when the update release image validation fails and the cluster is reporting errors.
+
+[IMPORTANT]
+====
+Use the `--force` option with caution. Only use `--force` if there are no other ways to update the cluster and if you are willing to accept the risks of waiving signature checks and other critical guards. 
```

Comment:
Alternatively, I think we could say "flag. eg, "Only use the `--force` flag. . ."

```suggestion
Use the `--force` option with caution. Only use the`--force` option if there are no other ways to update the cluster and if you are willing to accept the risks of waiving signature checks and other critical guards. 
```